torchreid/integration/nncf/accuracy_aware_training/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2021 Intel Corporation
```

Comment:
This license header looks outdated. We use
```
# Copyright (C) 2020-2021 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```
@LeonidBeynenson could you clarify this?